src/katgpucbf/utils.py

```diff
@@ -20,14 +20,19 @@
 import logging
 import signal
 from asyncio import get_event_loop
+from enum import Enum
+from typing import Optional, TypeVar
 
-from aiokatcp import DeviceServer
+import aiokatcp
 from katsdptelstate.endpoint import endpoint_list_parser
 
+_T = TypeVar("_T")
+_U = TypeVar("_U")
```

Comment:
I see why you've done `_T`, but is `_U` necessary for this PR? Or just future-proofing/insisting on a convention?

Reply:
You don't seem to use `_U` anywhere, so I'm guessing it is copy-pasted from aiokatcp.

Reply:
I'll have you know I typed it out manually while looking at the aiokatcp source!

I thought initially that I'd need both, but I guess I ended up not using the second one and I forgot to remove it.